extension-manifest-v2/src/classes/FoundBugs.js

```diff
@@ -309,20 +310,27 @@ class FoundBugs {
 	 * @param  {number} tab_id 	tab id
 	 * @return {Object}        	counts for blocked and allowed trackers
 	 */
-	getAppsCountByBlocked(tab_id) {
+	getTrackerCounts(tab_id, tabUrl) {
 		if (!this._init(tab_id)) {
 			return {
 				blocked: 0,
-				allowed: 0
+				allowed: 0,
+				found: 0,
 			};
 		}
 
 		const { blocked } = this._foundApps[tab_id].issueCounts;
 		const allowed = this._foundApps[tab_id].apps.length - blocked;
 
+		const adBlockingTrackers = getCommonData(tab_id, tabUrl).unidentifiedTrackers;
```

Comment:
Would feel more natural to explicitly list the third argument: 

const adBlockingTrackers = getCommonData(tab_id, tabUrl, false).unidentifiedTrackers;
const antiTrackingTrackers = getCommonData(tab_id, tabUrl, true).unidentifiedTrackers;

Took me first a while to understand what was the difference been both getCommonData calls.

Reply:
make sense. Fixed!